test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQOutputBindingTests.cs

```diff
@@ -1,36 +1,35 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Text;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Configuration;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions;
+using Microsoft.Azure.WebJobs.Extensions.RabbitMQ;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Moq;
 using Newtonsoft.Json;
-using RabbitMQ.Client;
 using Xunit;
 
-namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Tests
+namespace WebJobs.Extensions.RabbitMQ.Tests
 {
-    public class RabbitMQConfigurationTests
+    public class RabbitMQOutputBindingTests
     {
-        private static readonly IConfiguration _emptyConfig = new ConfigurationBuilder().Build();
-
         [Fact]
         public void Creates_Context_Correctly()
         {
             var options = new RabbitMQOptions { Hostname = "localhost", QueueName = "hello" };
             var loggerFactory = new LoggerFactory();
             var mockServiceFactory = new Mock<IRabbitMQServiceFactory>();
-            var config = new RabbitMQExtensionConfigProvider(new OptionsWrapper<RabbitMQOptions>(options), mockServiceFactory.Object, (ILoggerFactory)loggerFactory);
-            var attribute = new RabbitMQAttribute { Hostname = "localhost", QueueName = "queue" };
+            var mockNameResolver = new Mock<INameResolver>();
+            var config = new RabbitMQExtensionConfigProvider(new OptionsWrapper<RabbitMQOptions>(options), mockNameResolver.Object, mockServiceFactory.Object, (ILoggerFactory)loggerFactory);

```

Comment:
test with different host names i.e do not use localhost for options and attribute